analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/TokenTypeAnalyzerTest.cs

```diff
@@ -160,6 +173,14 @@ private void Verify(string fileName, ProjectType projectType, Action<IReadOnlyLi
                     });
         }
 
+        private static VerifierBuilder CreateBuilder(UtilityAnalyzerBase analyzer, string fileName, AnalyzerLanguage language, string testRoot) =>
+            new VerifierBuilder()
```

Comment:
Same comment as for the other test. We can create the analyzer inside this method.

Reply:
Cool.